README.md

```diff
@@ -23,11 +23,34 @@ This repository hosts the primary codebase for the Minsta app.
 
 This repository hosts the primary codebase for the [Minsta App](https://minsta.me), originally created and developed by [@microchipgnu](https://github.com/microchipgnu).
 
-All new Minsta instances should be derived from this repository.  
+All new Minsta instances should be derived from this repository.
 
 [Video Tutorial on Setting up](https://www.loom.com/share/d5a038fb341c40be9ae131dd82f199a80)
 
 
+## Pre-Setup - Proxy Contract
+
+1. You got to have a NEAR Contract where you add a proxy contract as a minter.
+2. The proxy contract is meant for let any user who is not a minter, to be able to mint images on Minsta.
+3. The Near Contract will be where the NFT images will be minted
```

Comment:
I am not sure I understand this sentence... dos this mean "the near contract supplied in step 1 will be the mint address of the image"?

I thought images were stored off-chain.